Gemfile

```diff
@@ -31,7 +31,7 @@ source 'https://rubygems.org'
 gem "rails", "~> 3.2.21"
 
 gem "coderay", "~> 1.0.5"
-gem "rubytree", "~> 0.8.3"
+gem "rubytree"
```

Comment:
Because people tend to use "bundle update" instead of "bundle (install)", we tend to restrict the versions.